### PR TITLE
PlottingGL: Added option for switching off ability to select multiple…

### PR DESCRIPTION
**Pull request by Alboin**
… axes. Made it the default option with single axis selection.

### CODE REVIEW

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp [hunk 1]
```diff
@@ -82,6 +82,7 @@ ParallelCoordinates::ParallelCoordinates()
           TransferFunction{
               {{0.0, vec4{1, 0, 0, 1}}, {0.5, vec4{1, 1, 0, 1}}, {1.0, vec4{0, 1, 0, 1}}}}}
 
+    , multipleAxisSelection_("multipleAxisSelection", "Multiple Axis Selection", false)
```
Comment (r-englund) [inline modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: Should it be off by default? This changes the old behavior

**Review by r-englund — APPROVED** [review]
Nice, have some local changes myself to "fix" this, was gonna create a PR but you were quicker :)

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp (hunk 1 above)
Comment (Alboin) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: That's the question! Would it be a more common case to make multiple selections?

**Comment by ResearchDaniel** [comment]
Nice, but should it be multiple options?
Multiple
Single
Off

**Review by ResearchDaniel — APPROVED** [review]

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp (hunk 1 above)
Comment (ResearchDaniel) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: I don't think so. I guess if one wants to highlight the selected axis in a scatter plot matrix or so. But perhaps it is best to have the more general as default anyway?

**Comment by Alboin** [comment]
Made it an OptionProperty with, Single, Multiple and None as alternatives, where Multiple is selected by default.

**Review by ResearchDaniel — APPROVED** [review]
Sweet

**Pull request merged**